src/components/NavBar/NavBar.jsx

```diff
@@ -38,7 +38,7 @@ const NavBar = () => {
   const [openMenu, setOpenMenu] = useState(false);
   const menuId = 'primary-search-account-menu';
 
-  // state for LogoutModal component
+  // state for Logout Modal component
```

Comment:
Don't think we need to change this comment. The name for the component is LogoutModal, after all.

Unless you would like to rephrase it to `state for the logout modal`